app/registries/foreman/plugin.rb

```diff
@@ -294,6 +294,7 @@ def role(name, permissions, description = '')
     # Usage:
     # add_resource_permissions_to_default_roles ['MyPlugin::FirstResource', 'MyPlugin::SecondResource'], :except => [:skip_this_permission]
     def add_resource_permissions_to_default_roles(resources, opts = {})
+      return if Foreman.in_setup_db_rake? || !permission_table_exists?
```

Comment:
Would it be better for this early return to go in RbacSupport, i.e. where the actual DB activity is going to happen?

Reply:
Although there's other early returns in this plugin code, so maybe here is best. :man_shrugging:

Reply:
If you look at the history of this file, that return was moved here from Rbac some time ago.